### PR TITLE
Add dropdown menu for start&stop container and test the menu

### PR DESCRIPTION
**Pull request by QiWang19**

### CODE REVIEW

**Comment by QiWang19** [comment]
@martinpitt  PTAL

File: src/Dropdown.jsx [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
+
+        return (
+            <div className="btn-group" >
+                <Select.Select initial={this.state.title} onChange={(event) => this.handleClick(event)}>
+                    {menuItems}
+                </Select.Select>
+            </div>
+        );
+    }
+}
```
Comment (martinpitt) [inline src/Dropdown.jsx]: I really wonder if it's that beneficial to wrap an already existing `Select` React component into this new one -- there's really no extra logic here, just some glue.

Comment (QiWang19) [reply src/Dropdown.jsx]: looks like ```div``` can keep the distance between this dropdown button and the commit button in its left. Delete this div now?

**Comment by KKoukiou** [comment]
This functionality is already implemented.

**Pull request closed without merging**